### DISCUSSION TITLE
User Stories

### DISCUSSION THREAD

**Discussion by graciegoheen** [dbt-labs/dbt-meshify#1] [General]
As a dbt user with a single dbt project, I want to unlock multi-project deployment by splitting my project into 2+ projects without a lot of manual development time. 
- I want to split my repository code, creating new repositories for net new projects
- I want to identify and update appropriate refs to cross-project refs
- I want to identify and create model groups and constraints (if they don't yet exist)
- I want to create dbt Cloud resources for net new projects

As a dbt user with multiple dbt projects, I want to unlock multi-project deployment deployment by merging my projects together appropriates without a lot of manual development time.
- I want to merge metadata about “sources” and models from projects
- I want identify and update appropriate sources / refs to cross-project refs (depending on if I'm using the "source hack" or "private package" approach)
- I want to identify and create model groups and constraints (if they don't yet exist)

**Comment by adrienneyager** [comment]
@graciegoheen This is a great enumeration! Thanks for putting it together to keep us on track. My only suggestion is that we move creating dbt cloud resources into a third milestone. This way, we can start by focusing on project carving and stitching :tm: locally, first.
